app/forms/NirmsNumberFormProvider.scala

```diff
@@ -14,14 +14,19 @@
  * limitations under the License.
  */
 
-package navigation
+package forms
 
-import play.api.mvc.Call
-import pages._
-import models.{Mode, UserAnswers}
+import forms.mappings.Mappings
+import play.api.data.Form
 
-class FakeNavigator(desiredRoute: Call) extends Navigator {
+import javax.inject.Inject
+
+class NirmsNumberFormProvider @Inject() extends Mappings {
+
+  def apply(): Form[String] =
+    Form(
+      "nirmsNumber" -> text("nirmsNumber.error.required")
+        .verifying(regexp("RMS-(GB|NI)-[0-9]{6}", "nirmsNumber.error.invalidFormat"))
```

Comment:
I believe the dashes should be optional

Reply:
Ah, will confirm with the design team then? The example looks a bit confusing in that case.